utils/play_with_units_in_klaytn/src/main/java/Boilerplate.java

```diff
@@ -0,0 +1,110 @@
+    }
+
+    public static void loadEnv() {
+        Dotenv env;
+        String workingDirectory = System.getProperty("user.dir");
+        Path workingDirectoryPath = Paths.get(workingDirectory);
+        String projectRootDirectory = "caver-java-examples";
+        String currentDirectoryName = workingDirectoryPath.getName(workingDirectoryPath.getNameCount() - 1).toString();
+        String envDirectory = currentDirectoryName.equals(projectRootDirectory) ?
+                workingDirectoryPath.toString() :
+                workingDirectoryPath.getParent().getParent().toString();
+
+        // Read `/path/to/caver-java-examples/.env` file.
+        env = Dotenv.configure().directory(envDirectory).load();
+
+        nodeApiUrl = nodeApiUrl.equals("") ? env.get("NODE_API_URL") : nodeApiUrl;
+        accessKeyId = accessKeyId.equals("") ? env.get("ACCESS_KEY_ID") : accessKeyId;
+        secretAccessKey = secretAccessKey.equals("") ? env.get("SECRET_ACCESS_KEY") : secretAccessKey;
+        chainId = chainId.equals("") ? env.get("CHAIN_ID") : chainId;
+        senderAddress = senderAddress.equals("") ? env.get("SENDER_ADDRESS") : senderAddress;
+        senderPrivateKey = senderPrivateKey.equals("") ? env.get("SENDER_PRIVATE_KEY") : senderPrivateKey;
+        recipientAddress = recipientAddress.equals("") ? env.get("RECIPIENT_ADDRESS") : recipientAddress;
+    }
+
+    public static void run() {
+        HttpService httpService = new HttpService(nodeApiUrl);
+        httpService.addHeader("Authorization", Credentials.basic(accessKeyId, secretAccessKey));
+        httpService.addHeader("x-chain-id", chainId);
+        Caver caver = new Caver(httpService);
+
+        SingleKeyring senderKeyring = caver.wallet.keyring.create(senderAddress, senderPrivateKey);
+        caver.wallet.add(senderKeyring);
+
+        // Because a field "value" always interprets its value as a unit "peb",
+        // you must take care what is the actual value when you sending some KLAY.
+        ValueTransfer vt = caver.transaction.valueTransfer.create(
+                TxPropertyBuilder.valueTransfer()
+                        .setFrom(senderKeyring.getAddress())
+                        .setTo(recipientAddress)
+                        .setGas(BigInteger.valueOf(25000))
+                        .setValue(BigInteger.ONE)
+        );
+        // Example-1: Sending 0.5 KLAY to recipient
+        // 1 KLAY is actually 10^18(=1000000000000000000) peb. So if you want send 0.5 KLAY,
+        // option-1: Set actual peb value directly to ValueTransfer transaction instance.
+        vt.setValue(BigInteger.valueOf(500000000000000000L)); // 5 * (10^17)
+        System.out.println("Example-1) The value what we set using option-1 is " + vt.getValue());
+
+        // option-2 (Recommended): option-1 is very cumbersome and difficult to read. Let's use more elegant way.
+        String KLAY = caver.utils.convertToPeb(BigDecimal.valueOf(5), Utils.KlayUnit.KLAY);
```

Comment:
Please use lower case.

Reply:
Thanks for your feedback. I will. 👍

Reply:
@sirano11 
I updated it here.
Please review this [file](https://github.com/klaytn/caver-java-examples/blob/4757e5b9399fd447a242815f1cc6eefc05a87335/utils/play_with_units_in_klaytn/src/main/java/Boilerplate.java) :)